File: app/src/main/java/cn/intret/app/picgo/model/NotEmptyException.java
package cn.intret.app.picgo.model;

/**
 * Created by intret on 2017/9/23.
 */

public class NotEmptyException extends Exception {

    public NotEmptyException() {
    }

    public NotEmptyException(String message) {
        super(message);
    }

    public NotEmptyException(String message, Throwable cause) {
        super(message, cause);
    }

    public NotEmptyException(Throwable cause) {
        super(cause);
    }

    public NotEmptyException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: app/src/main/java/cn/intret/app/picgo/model/NotEmptyException.kt
package cn.intret.app.picgo.model

import android.annotation.TargetApi
import android.os.Build

/**
 * Created by intret on 2017/9/23.
 */

class NotEmptyException : Exception {

    constructor()

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)

    @TargetApi(Build.VERSION_CODES.N)
    constructor(message: String, cause: Throwable, enableSuppression: Boolean, writableStackTrace: Boolean) : super(message, cause, enableSuppression, writableStackTrace)
}
